src/main/webapp/resources/js/i18n.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * On load, get all the internationalization strings and combine them into one object
```

Comment:
Seeing this is a pretty key part of the system, can you bump up the comments in this file as well please?

Reply:
Updated in 38523bdd264be4bef74409991d2dab219fd98a2d